### PR TITLE
Created calender

### PR DESCRIPTION
**Pull request by AyolaM**

### CODE REVIEW

**Comment by AyolaM** [comment]
first pr

File: index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  </head>
+  <body>
+    <div class="calender">
+      <h1>Planner</h1>
+      <h2>Calender</h2>
+
+      <div class="month">
+        <ul>
+          <li class="prev">&#10094;</li>
+          <li class="next">&#10095;</li>
```
Comment (schuster-eb) [inline index.html:18]: If these are supposed to be buttons please nest `<button></button>` inside those `li`s

File: index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        <li>28</li>
+        <li>29</li>
+        <li>30</li>
+      </ul>
+    </div>
+    <div class="todo">
+      <h2>To Do List</h2>
+
+      <input type="text" id="text-todo" />
+      <button onclick="alert('Added')">Add to do</button>
```
Comment (schuster-eb) [inline index.html:70]: Personal preference, I'm not a fan of `onclick` would prefer `window.addEventListener('click', ()=>{})`

**Comment by schuster-eb** [comment]
Looks good. My comments can be follow on tasks

**Review by schuster-eb — APPROVED** [review]

**Pull request merged**